### DISCUSSION TITLE
Export the `UnistylesValues` type

### DISCUSSION THREAD

**Discussion by efoken** [jpudysz/react-native-unistyles#128] [Feature Requests]
We're planning to use Unistyles for a customer project where we have APIs like `styled` and an `sx` prop. Therefore it would be super helpful to use the `UnistylesValues` type from `src/types/stylesheet.ts`. Currently I'm extracting it like this:

```ts
type Style = Exclude<
  ReturnType<Extract<Parameters<typeof createStyleSheet>[0], Function>>[string],
  Function
>;
```

Which seems a bit unstable for me 😅

**Comment by jpudysz** [comment]
Ouch, that must hurt 😅

I can export it, I dont see any problem. 

PS. To speed things up, I will happily accept a PR, as I’m on vacation and only opening my computer once every couple of days 🙏

**Comment by efoken** [comment]
@jpudysz https://github.com/jpudysz/react-native-unistyles/pull/129

**Reply by jpudysz** [reply]
Merged and released in 2.0.2. Thanks!
